content/en/dashboards/ddsql_editor/reference/expressions_and_operators.md

```diff
@@ -64,4 +64,50 @@ DDSQL supports the `BETWEEN` keyword such that `a BETWEEN x AND y` is equivalent
 | IS NULL | Returns true for each row that is null |
 
 [1]: /dashboards/ddsql_editor/reference/tags/
-[2]: https://www.postgresql.org/docs/current/functions-comparison.html
\ No newline at end of file
+[2]: https://www.postgresql.org/docs/current/functions-comparison.html
+
+## CASE
```

Comment:
While `CASE` and `CAST` begin with keywords like a statement or clause, they are technically expressions in SQL - they can be used as (or inside) the `expressions` placeholder in the syntax described under `SELECT` in the  "Statements" page.